### ISSUE TITLE
Dataset URIs in metadata and in main tll dumps are not connected

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/dipper#545]
in panther.ttl

```
MonarchData:panther.ttl a owl:Ontology ;
    owl:versionIRI MonarchArchive:panther-2017-02-10-13-07.ttl ;
    owl:versionInfo "panther-2017-02-10-13-07" .
```

in panther_dataset.ttl:

```
@prefix : <https://monarchinitiative.org/> .

<https://monarchinitiative.org/2017-10-04T19:13:08> dct:isVersionOf :panther2017-10-05 ;
    dct:issued "2017-10-04T19:13:08"^^xsd:dateTime ;
    pav:version "2017-10-04T19:13:08" .

:panther a dctypes:Dataset ;
    dct:identifier "panther" ;
    dct:issued "2017-10-05" ;
    dct:license <http://www.pantherdb.org/terms/disclaimer.jsp> ;
    dct:title "Protein ANalysis THrough Evolutionary Relationships" ;
    dcat:accessURL <ftp://ftp.pantherdb.org/ortholog/current_release/Orthologs_HCOP.tar.gz>,
        <ftp://ftp.pantherdb.org/ortholog/current_release/RefGenomeOrthologs.tar.gz> ;
    foaf:page <http://pantherdb.org/> .

:panther2017-10-05 dct:isVersionOf :panther ;
    pav:version "2017-10-05" .
```

note that these 2 graphs don't link up! We need a triple connecting MonarchData:panther.ttl to MonarchData:panther, stating that the former is some formatted version of the latter logical resource.

Related: when we store this in Solr, we use our MonarchData:panther.ttl as the source. The actual linkage to the original source is not AFAICT apparent. I think we should use MonarchData:panther

**Comment by TomConlin** [comment]
Definitely agree about not hard coding  the extension. just about to make a PR that makes generating non turtle files easier and safer and includes nquads.  And likewise I want the IRI for the named graph in the quads to be the IRI for the `foo_dataset.ttl` resource.

**Comment by justaddcoffee** [comment]
addressed by #809
